lib/pxapi.js

```diff
@@ -49,7 +49,8 @@ function callServer(pxCtx, callback) {
             http_method: pxCtx.httpMethod,
             risk_mode: riskMode,
             module_version: config.MODULE_VERSION,
-            cookie_origin: pxCtx.cookieOrigin
+            cookie_origin: pxCtx.cookieOrigin,
+            risk_cookie_names: pxCtx.risk_cookie_names
```

Comment:
should be `pxCtx. riskCookieNames`

Reply:
how did work when you tested it?

Reply:
It worked because I tested in with the correct name. Don't knwo how the name switch has happened